include/VArrayProxy.hpp

```diff
@@ -18,29 +18,28 @@ struct VArrayProxy
 	static VArrayProxy<T>::ConstPtr create(std::shared_ptr<const VArray> src)
 	{ return VArrayProxy<T>::ConstPtr(new VArrayProxy<T>(std::const_pointer_cast<VArray>(src))); }
 
-	VArrayProxy<T>::Ptr clone() const
-	{ return VArrayProxy<T>::create(src->clone()); }
-
 	VArray::Ptr untyped() { return src; };
 	VArray::ConstPtr untyped() const { return src; };
 
-	void copyFrom(const T* srcRaw, std::size_t count) { src->copyFrom(srcRaw, count); }
+	void copyFrom(const T* srcRaw, std::size_t count) { src->setData(srcRaw, count); }
```

Comment:
I'd suggest having the same names of functions for both - VArray and VArrayProxy.